### PR TITLE
Add ccc and patient name to manifest samples table and flu sample types

### PR DESCRIPTION
**Pull request by Omoshlawi**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

- Added patient name and ccc to to manifest sample for easy verification
- Added flu sample types

## Screenshots
Screenshot 2024-12-03 at 08 53 29

https://github.com/user-attachments/assets/4a21b945-8f2c-4e64-b882-8e0605456932

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-lab-manifest-app/src/forms/sample-delete-confirm-dialog.modal.tsx [hunk 1]
```diff
@@ -30,7 +32,11 @@ const SampleDeleteConfirmDialog: React.FC<SampleDeleteConfirmDialogProps> = ({ o
 
   const headers = [
     {
-      header: t('patientIdentifier', 'Patient Identifier'),
+      header: t('patient', 'Patient'),
+      key: 'patient',
+    },
+    {
+      header: t('cccKDODNumber', 'CCC/KDOD Number'),
```
Comment (makombe) [inline packages/esm-lab-manifest-app/src/forms/sample-delete-confirm-dialog.modal.tsx:39]: Is the KDOD number same as ccc number? I can remember cc @njorocs

**Review by makombe — APPROVED** [review]
Thanks @Omoshlawi

File: packages/esm-lab-manifest-app/src/config-schema.ts [hunk 2]
```diff
@@ -28,8 +28,14 @@ export const configSchema = {
       { sampleType: 'DBS', labManifestType: ['1'] },
     ],
   },
+  cccNumberIdentifierType: {
```
Comment (donaldkibet) [inline packages/esm-lab-manifest-app/src/config-schema.ts:31]: Should this just have the ccc identifier type or should it be an array of identifiers to be used here

**Review by donaldkibet — CHANGES_REQUESTED** [review]

File: packages/esm-lab-manifest-app/src/config-schema.ts (hunk 2 above)
Comment (Omoshlawi) [reply packages/esm-lab-manifest-app/src/config-schema.ts:31]: For the manifest samples, the required identifier of interest was only ccc for verification

File: packages/esm-lab-manifest-app/src/forms/sample-delete-confirm-dialog.modal.tsx (hunk 1 above)
Comment (Omoshlawi) [reply packages/esm-lab-manifest-app/src/forms/sample-delete-confirm-dialog.modal.tsx:39]: just saw that we have kdod identifier type, should it check for ccc, if not found then look for kdod identifier for the patient?

File: packages/esm-lab-manifest-app/src/config-schema.ts (hunk 2 above)
Comment (Omoshlawi) [reply packages/esm-lab-manifest-app/src/config-schema.ts:31]: just found out we have kdod identifier type as well, so based on teams recommendation on bellow quoted question, i might change it as you have recommended

> just saw that we have kdod identifier type, should it check for ccc, if not found then look for kdod identifier for the patient?and if all are available which one should be displayed? cc @PatrickWaweru @donaldkibet

File: packages/esm-lab-manifest-app/src/forms/sample-delete-confirm-dialog.modal.tsx (hunk 1 above)
Comment (PatrickWaweru) [reply packages/esm-lab-manifest-app/src/forms/sample-delete-confirm-dialog.modal.tsx:39]: It depends on the system type. If it is a KDOD system, display KDOD. Else, display CCC.

File: packages/esm-lab-manifest-app/src/config-schema.ts (hunk 2 above)
Comment (PatrickWaweru) [reply packages/esm-lab-manifest-app/src/config-schema.ts:31]: It depends on the system type. If it is a KDOD system, display KDOD. Else, display CCC.

Comment (donaldkibet) [reply packages/esm-lab-manifest-app/src/config-schema.ts:31]: Thanks @PatrickWaweru, @Omoshlawi we will have to check the global property to determine which identifier to show.

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**